Tasks/AzureFileCopyV4/Utility.ps1

```diff
@@ -214,6 +214,7 @@ function Upload-FilesToAzureContainer
 
         $blobPrefix = $blobPrefix.Trim()
         $containerURL = [string]::Format("{0}/{1}/{2}", $blobStorageEndpoint.Trim("/"), $containerName, $blobPrefix).Trim("/")
+        $containerURL = $containerURL + "/"
```

Comment:
I think a better solution would be not to trim `/` at L216 at all. Instead of calling it "Blob Prefix", call it "Destination". Keep it as is and pass it to `azcopy`. Leave its interpretation to `azcopy`: If `20220328.38/yum/`, treat it as virtual directory. If `20220328.38/yum`, treat it as blob name.